packages/app-store/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -187,116 +225,136 @@ export default function RoutingForms({
+                        const firstItem = forms[0].form;
+                        const lastItem = forms[forms.length - 1].form;
+
                         return (
-                          <ListLinkItem
-                            key={form.id}
-                            href={appUrl + "/form-edit/" + form.id}
-                            heading={form.name}
-                            disabled={readOnly}
-                            subHeading={description}
-                            className="space-x-2 rtl:space-x-reverse"
-                            actions={
-                              <>
-                                {form.team?.name && (
-                                  <div className="border-r-2 border-neutral-300">
-                                    <Badge className="ltr:mr-2 rtl:ml-2" variant="gray">
-                                      {form.team.name}
-                                    </Badge>
-                                  </div>
-                                )}
-                                <FormAction
-                                  disabled={readOnly}
-                                  className="self-center"
-                                  action="toggle"
-                                  routingForm={form}
-                                />
-                                <ButtonGroup combined>
-                                  <Tooltip content={t("preview")}>
-                                    <FormAction
-                                      action="preview"
-                                      routingForm={form}
-                                      target="_blank"
-                                      StartIcon={ExternalLink}
-                                      color="secondary"
-                                      variant="icon"
-                                    />
-                                  </Tooltip>
-                                  <FormAction
-                                    routingForm={form}
-                                    action="copyLink"
-                                    color="secondary"
-                                    variant="icon"
-                                    StartIcon={LinkIcon}
-                                    tooltip={t("copy_link_to_form")}
-                                  />
+                          <div className="group flex w-full max-w-full items-center justify-between overflow-hidden">
+                            {!(firstItem && firstItem.id === form.id) && (
+                              <button
+                                className="bg-default text-muted hover:text-emphasis border-default hover:border-emphasis invisible absolute left-[5px] -ml-4 -mt-4 mb-4 hidden h-6 w-6 scale-0 items-center justify-center rounded-md border p-1 transition-all group-hover:visible group-hover:scale-100 sm:ml-0 sm:flex lg:left-[36px]"
```

Comment:
can we create a common component for these Arrow Up and Arrow Down button ? It's used in /event-types, routing forms and workflow